buildSrc/src/main/resources/checkstyle.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+    "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+    "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+    <property name="charset" value="UTF-8" />
+
+    <module name="SuppressionFilter">
+        <property name="file" value="${suppressions}" />
+    </module>
+
+    <!-- Checks Java files and forbids empty Javadoc comments -->
+    <module name="RegexpMultiline">
+        <property name="format" value="\/\*[\s\*]*\*\/"/>
+        <property name="fileExtensions" value="java"/>
+        <property name="message" value="Empty javadoc comments are forbidden"/>
+    </module>
+
+    <module name="TreeWalker">
+        <!-- Its our official line length! See checkstyle_suppressions.xml for the files that don't pass this. For now we
+          suppress the check there but enforce it everywhere else. This prevents the list from getting longer even if it is
+          unfair. -->
+        <module name="LineLength">
+            <property name="max" value="100"/>
```

Comment:
Given the choice, I'd opt for longer lines. I think it's a pretty rare screen these days that won't accommodate line lengths of 120 or more.

Reply:
@danhermann yea but diffing (and eventually reading too) becomes a pain with more and more information per line :) => I'm a big fan of `100` here tbh :D

Reply:
Sure, that's fine. I just threw it out there in case no one had other preferences.